l3.S

```diff
@@ -1,8 +1,7 @@
-.globl l3_log_fast // l3_log_fast(msg)
+.globl l3__log_fast // l3__log_fast(msg)
 .extern l3_log
-.extern mytid
 
-l3_log_fast:
+l3__log_fast:
```

Comment:
@gregthelaw - I need your help to rework this assembly logic to support the extended logging interface I am trying to support, as defined in `l3.h` in this commit:

```
     82 #ifdef L3_LOC_ENABLED
     83 #define l3_log_fast(msg) l3__log_fast(__LOC__, (msg))
     84 #else   // L3_LOC_ENABLED
     85 #define l3_log_fast(msg) l3__log_fast((msg), L3_ARG_UNUSED)
     86 #endif  //

[...]

     97 #ifdef L3_LOC_ENABLED
     98 void l3__log_fast(const char *msg);
     99 #else
    100 void l3__log_fast(const char *msg, uint32_t loc);
    101 #endif  // L3_LOC_ENABLED
```

How do I rework the assembly code to save-off the 4-byte loc value?

Can you implement this change in this dev-branch `gapisback/fast_log-LOC-support` and commit the delta? 

I will rework test cases with that assembly support.

Reply:
Ignore L83 for now. Once the assembly is working, I will change all interfaces to become:

`l3__log_fast(const char *msg, uint32_t loc);`

i.e., the LOC-ID can be the 2nd arg to the call, if that's better for the assembly.